modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -225,6 +226,26 @@ public void deleteObjectEntry(
 		_objectEntryService.deleteObjectEntry(objectEntry.getObjectEntryId());
 	}
 
+	public void deleteObjectRelationshipMappingTableValues(
```

Comment:
@sergiojimcos I think a better name for this is `disassociateRelatedModels`. `deleteObjectRelationshipMappingTableValues` is too low level and you're anyway delegating to `objectRelatedModelsProvider.disassociateRelatedModels`.

Reply:
but @4lejandrito , in that case, I guess the opposite method `addObjectRelationshipMappingTableValues` should be also renamed ???

Reply:
I'll be create a new Technical Dev task for put it in the ObjectRelatedModelProvider